src/hooks/engine.rs

```diff
@@ -973,6 +1003,40 @@ pub mod exported {
         })
     }
 
+    #[cfg_attr(
+        not(feature = "bxt-compatibility"),
+        export_name = "R_DrawSequentialPoly"
+    )]
+    pub unsafe extern "C" fn my_R_DrawSequentialPoly(
+        surf: *mut c_void,
+        face: *mut c_int,
+    ) -> *mut c_void {
+        abort_on_panic(move || {
+            let marker = MainThreadMarker::new();
+
+            if wallhack::is_active(marker) {
```

Comment:
Also BXT has a useful comment on this, would be worth preserving it here: https://github.com/YaLTeR/BunnymodXT/blob/2c4bd86762b5efd7433413bdaff4a584ce92d89b/BunnymodXT/modules/HwDLL.cpp#L4476